Documentation/project-docs/developer-guide.md

```diff
@@ -53,17 +51,17 @@ clean -all
 build
 ```
 
-- Building in debug mode for platform x64
+- Building in release mode for platform x64 (restore and build are implicit here as no actions are passed in)
 ```
-build -debug /p:ArchGroup=x64
+build -c Release -arch x64
 ```
 
-- Building the src and then building and running the tests
+- Building the src and build and run tests
 ```
-build -includetests
+build & build -test -buildtests
```

Comment:
is & necessary, does this work? `build -build -buildtests -build`

Reply:
Ah, but then you need to pass -restore?

Reply:
you mean `build -build -buildtests -test`? no multiple actions are not supported.

Reply:
that command above an be executed in the following ways:

1. build -restore -build -buildtests & build -test
2. build & build -test -buildtests

Reply:
Should we support multiple actions? Just call the targets in the order of the actions? Or that just over complicate stuff?

Reply:
that's just not supported by arcade ;) arcade would need to call into our build.proj N times (N = number of actions).